src/adam_impact_study/conversions.py

```diff
@@ -162,75 +119,144 @@ def sorcha_output_to_od_observations(sorcha_observations_df):
+            object_id=[obj for i in range(len(object_obs))],
             coordinates=coordinates_sorted,
             observers=Observers.from_code("X05", coordinates_sorted.time),
+            photometry=photometry,
         )
 
+        if od_observations is None:
+            od_observations = od_observation
+        else:
+            od_observations = qv.concatenate([od_observations, od_observation])
+
     return od_observations
 
 
-def sorcha_df_to_fo_input(sorcha_df, fo_file_name):
+def od_observations_to_fo_input(od_observations, fo_file_name):
     """
-    Convert a Sorcha DataFrame to a Find_Orb input file.
+    Convert an Observations object into a Find_Orb input file.
 
     Parameters
     ----------
-    sorcha_df : pandas.DataFrame
-        DataFrame containing Sorcha data with relevant columns:
-        - "ObjID": Object ID
-        - "fieldMJD_TAI": Observation time in MJD (TAI)
-        - "RA_deg": Right Ascension in degrees
-        - "Dec_deg": Declination in degrees
-        - "astrometricSigma_deg": Astrometric uncertainty in degrees
-        - "trailedSourceMag": Trailed source magnitude
-        - "trailedSourceMagSigma": Uncertainty in trailed source magnitude
-        - "optFilter": Optical filter used
+    od_observations : qv.Table
+        Observations object containing observations to be converted.
 
     fo_file_name : str
         Name of the Find_Orb input file to be created.
 
+    object_id : str
+        Object ID of orbit connected to observations.
+
     Returns
     -------
     fo_file_name : str
         Path to the generated Find_Orb input file.
     """
     with open(fo_file_name, "w") as w:
-        w.write("trkSub|stn|obsTime|ra|dec|rmsRA|rmsDec|mag|rmsMag|band\n")
-        for index, row in sorcha_df.iterrows():
-            time_tai = Timestamp.from_mjd([row["fieldMJD_TAI"]], scale="tai")
-            time_utc = time_tai.rescale("utc")
+        w.write("trkSub|stn|obsTime|ra|dec|rmsRA|rmsDec\n")  # |mag|rmsMag|band
+        for obs in od_observations:
+            sigmas = obs.coordinates.covariance.sigmas
+            time_utc = obs.coordinates.time
             time = time_utc.to_astropy()
             w.write(
-                f"{row['ObjID']}|X05|{time.isot[0]}|{row['RA_deg']}|"
-                f"{row['Dec_deg']}|"
-                f"{float(row['astrometricSigma_deg'])*3600}|"
-                f"{float(row['astrometricSigma_deg'])*3600}|"
-                f"{row['trailedSourceMag']}|"
-                f"{row['trailedSourceMagSigma']}|"
-                f"{row['optFilter']}\n"
+                f"{obs.object_id}|X05|{time.isot[0]}|{obs.coordinates.lon[0]}|"
+                f"{obs.coordinates.lat[0]}|"
+                f"{format(sigmas[0][1]*3600, '.5f')}|"
+                f"{format(sigmas[0][2]*3600, '.5f')}|"
+                f"{obs.photometry.mag}|"
+                f"{obs.photometry.mag_sigma}|"
+                f"{obs.photometry.filter}\n"
```

Comment:
I think we should use our (new) existing ADES utility for this. That way we have a central place where time and deg/arcsecond versions are happening and we can unit test that one spot. It would look something like this I think

```python
ades_observations = ADESObservations.from_kwargs(
    provID=observations.object_id,
    trkSub=observations.object_id,
    obsSubID=observations.obs_id,
    obsTime=observations.coordinates.time,
    ra=observations.coordinates.ra,
    dec=observations.coordinates.dec,
    rmsRA=observations.coordinates.covariance.sigmas[0][1],
    rmsDec=observations.coordinates.covariance.sigmas[0][2],
    mag=observations.photometry.mag,
    rmsMag=observations.photometry.mag_sigma,
    band=observations.photometry.filter,
    stn=observations.observers.name,
    mode="unspecified",
    astCat="unspecified",
    remarks="unspecified",
)

obs_contexts = {
    "X05": ObsContext(
        observatory=ObservatoryObsContext(
            mpcCode="X05", name="Vera C. Rubin Observatory - LSST"
        ),
        submitter=SubmitterObsContext(
            name="None"
        ),
        observers=[],
        measurers=[],
        telescope=TelescopeObsContext(
            name="LSST Camera", design="Reflector", detector="CCD", aperture=8.4
        ),
    )
}

ades_string = ADES_to_string(ades_observations, obs_contexts)
w.write(ades_string)
```

Obviously that's untested, but I believe it should work. Good for us to know if adam core is producing ADES that find_orb can consume.